src/components/pages/membership_form/Payment.vue

```diff
@@ -113,4 +114,14 @@ watch( minimumAmount, async ( newMinimumAmount ) => {
 		fee.value = '';
 	}
 } );
+
+const feeErrorMessage = computed<String>( () => {
```

Comment:
```suggestion
const feeErrorMessage = computed<string>( () => {
```
the string type in TypeScript is usually written in lower case, only some legacy Vue/Preact code has a `String` type for props.

I wonder why TypeScript doesn't complain here, since the `t()` function returns a `TranslateResult`, not a `string`. Or is that the reason why you call `feeErrorMessage.toString`?